bug_report_page.php

```diff
@@ -172,7 +172,7 @@
 	$f_steps_to_reproduce	= gpc_get_string( 'steps_to_reproduce', config_get( 'default_bug_steps_to_reproduce' ) );
 	$f_additional_info		= gpc_get_string( 'additional_info', config_get( 'default_bug_additional_info' ) );
 	$f_view_state			= gpc_get_int( 'view_state', (int)config_get( 'default_bug_view_status' ) );
-	$f_due_date				= gpc_get_string( 'due_date', '' );
+	$f_due_date				= gpc_get_string( 'due_date', strtotime( config_get( 'due_date_default' ) ) );
```

Comment:
strtotime() won't handle the case where due_date_default is set to empty string properly.  Please add date_strtotime() in core/date_api.php which does the following:
- If input is empty string, return date_get_null()
- otherwise return strtotime() of the input value.

Then you can call it here.